### PR TITLE
Fix awaiting an array of custom attributes

### PR DESCRIPTION
**Pull request by tim-stasse**
- Also fixed errors being caught, and not stopping the process when encountered (by re-throwing the error after logging it).
- Also formatted file using prettier (Sorry about the diff being a bit noisy, I'm using vscode with auto format on save turned on).

### CODE REVIEW

**Comment by tim-stasse** [comment]
Fixes #6

File: index.js [hunk 1]
```diff
@@ -133,72 +199,105 @@ class CognitoAddCustomAttributesPlugin {
...
+
+              // if we run more than one at a time, there is a chance for a race condition on creating
+              // the customAttribute on the userPool the first time
+              await this.processCognitoCustomAttributeMapping(
+                stack,
+                mappingItem
+              );
+            })()
+          )
+        );
```
Comment (tim-stasse) [inline index.js]: This is the main fix, replacing the `forEach` for a `map` returning a promise (the async function is invoked inline) and wrapping it with `Promise.all`.

Comment (jmaleonard) [reply index.js]: LGTM

**Comment by jmaleonard** [comment]
Thanks for the fix. I am going to accept this change. Can you please include your .prettierrc file here so that we can include it in this repo?

**Pull request merged**